<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="verticalFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>811</width>
      <height>811</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="DisplayView">
       <item>
        <widget class="QGraphicsView" name="sourceView"/>
       </item>
       <item>
        <widget class="QGraphicsView" name="targetView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ImageDisplay">
       <item>
        <widget class="QRadioButton" name="source">
         <property name="text">
          <string>Source</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="target">
         <property name="text">
          <string>target</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="processLayout">
       <item>
        <widget class="QRadioButton" name="IHS">
         <property name="text">
          <string>IHS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="MEAN">
         <property name="text">
          <string>Mean</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="BROVEY">
         <property name="text">
          <string>Brovey</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ESRI">
         <property name="text">
          <string>Esri</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="process">
       <property name="text">
        <string>Process</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QLabel" name="percent_3">
           <property name="text">
            <string>Weight: 0.00</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QSlider" name="weight_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSlider" name="weight_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="percent_2">
           <property name="text">
            <string>Weight: 0.00</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="percent_4">
           <property name="text">
            <string>Weight: 0.00</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QSlider" name="weight_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QSlider" name="weight_1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="percent_1">
           <property name="text">
            <string>Weight: 0.00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="outputView">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>FILE</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>VIEW</string>
    </property>
    <addaction name="actionzoomIn"/>
    <addaction name="actionzoomOut"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="viewMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionzoomIn">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionzoomOut">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
